src/main/webapp/app/app-routing.module.ts

```diff
@@ -52,6 +52,11 @@ const LAYOUT_ROUTES: Routes = [navbarRoute, ...errorRoute];
                     path: 'courses/:courseId/file-upload-exercises/:exerciseId',
                     loadChildren: () => import('./exercises/file-upload/participate/file-upload-participation.module').then((m) => m.ArtemisFileUploadParticipationModule),
                 },
+                // ===== EXAM =====
+                {
+                    path: 'courses/:courseId/exam',
```

Comment:
There will be the possibility to create multiple exams per course. Therefore we should add an `:examId` here. Even if the student gets a student exam instead of the exam we should still leave `exam` in the route as you did here.